usaspending_api/common/api_request_utils.py

```diff
@@ -492,9 +492,9 @@ def build_response(self):
 class ResponsePaginator:
     @staticmethod
     def get_paged_data(data_set, page=1, page_limit=100, request_parameters={}):
-        if 'limit' in request_parameters:
+        if request_parameters.get('limit') is not None:
```

Comment:
Yeah, you're right...the previous version should still work and is probably more readable. I just got swept away by the change from a standard dict back to the QueryDict.